src/NzbDrone.Core/Download/Clients/Flood/Flood.cs

```diff
@@ -143,7 +148,16 @@ public override IEnumerable<DownloadClientItem> GetItems()
                     item.Status = DownloadItemStatus.Paused;
                 }
 
-                item.CanMoveFiles = item.CanBeRemoved = false;
+                if (item.Status == DownloadItemStatus.Completed)
+                {
+                    // Grab cached seedConfig
+                    var seedConfig = _downloadSeedConfigProvider.GetSeedConfiguration(item.DownloadId);
+
+                    if (item.SeedRatio >= seedConfig.Ratio)
```

Comment:
check `seedConfig != null &&`

Reply:
Also add comment that SeedTime isn't supported. 
Maybe add that to the Message too?

Reply:
A `dateFinished` field is going to be added in newer versions of Flood, so I revised the patch to support that.